engine/installation/linux/docker-ce/centos.md

```diff
@@ -139,7 +139,7 @@ from the repository.
     > source packages as well, omit the `.x86_64` from the package name.
 
     ```bash
-    $ yum list docker-ce.x86_64  --showduplicates |sort -r
+    $ yum list docker-ce.x86_64  --showduplicates | sort -r
```

Comment:
I see netlify failed (can't see the reason), but wondering if it's related to this change, and another case of Jekyll seeing the pipe as a Markdown table